src/main/steps/application-within-proceedings/download-form/content.ts

```diff
@@ -0,0 +1,106 @@
+import { AWPApplicationType } from '../../../app/case/definition';
+import { TranslationFn } from '../../../app/controller/GetController';
+import { FormContent } from '../../../app/form/Form';
+import { getCasePartyType } from '../../../steps/prl-cases/dashboard/utils';
+import { getApplicationDetails } from '../utils';
+
+export const en = {
+  title: 'Download the form',
+  explainRequest: 'You can explain your request by completing and uploading the form',
+  nextSteps: 'Next steps',
+  line1: 'Download the form',
+  from: 'from',
+  govUk: 'GOV.UK (opens in a new tab)',
```

Comment:
Again make sure contents are in array and not split as lines

Reply:
Changed to use array with whole lines